authorinomanifests/authorino.yaml

```diff
@@ -605,31 +705,99 @@ spec:
         control-plane: controller-manager
     spec:
       containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=10
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
       - args:
         - --metrics-addr=127.0.0.1:8080
         - --enable-leader-election
         command:
         - /manager
-        image: quay.io/3scale/authorino:latest
         env:
-          - name: AUTHORINO_SECRET_LABEL_KEY
-            value: "secret.kuadrant.io/managed-by"
+        - name: WATCH_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/3scale/authorino:latest
```

Comment:
I would pin to an specific tag, otherwise you never know if deployed authorino matches with deployed manifests